### DISCUSSION TITLE
Sorting products manually

### DISCUSSION THREAD

**Discussion by marnauortega** [swellstores/community#343] [Help] [answered]
When querying for products in graphql I am able to sort by name and price, but how do I go about sorting them according to the manual order that I set in the dashboard? What sort string should I use?

Thanks a lot

**Comment by sags95** [comment, marked as answer]
Hey @marnauortega, in this case you would not need to sort actually, but just fetch all products. Unless I'm misunderstanding what you mean by manual order set in the dashboard.

So something like this:
```
query getAllProducts {
  products(limit: 25, page: 1, ) {
    results {
      id
      name
      slug
      price
      currency
      categories {
        slug
        topId
      }
      }
    }
  }
```

**Reply by marnauortega** [reply]
Oh, you are absolutely right. I was adding a wrong `sort` parameter, and that was preventing the order I had set in the dashboard to take effect. Thank you!
